temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message EndpointSpec {
+    // The name of the endpoint. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // Indicates where the endpoint should forward received nexus requests to. 
+    EndpointTargetSpec target_spec = 2;
+
+    // The set of authorization policies for the endpoint. Each request's caller
+    // must match with at least one of the specs to be accepted by the endpoint.
+    // This field is mutable.
+    repeated EndpointAuthorizationSpec authorization_specs = 3;
```

Comment:
I'm now thinking maybe instead of calling these authorization specs, call them policies.
We can then add rate limiting rules here too.
So I can express constraints like `allow this namespace but limit it to 100 RPS` in the future.

Reply:
changed to PolicySpec